frontend/smart-kickers-game/src/components/Game/GameStatistics/NumberOfShots.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './GameStatistics.css';
+
+function NumberOfShots(statistics) {
+  return (
+    <>
+      <div className="table-item">{statistics.blue}</div>
```

Comment:
Are you sure that `statistics` will exist every time?

Reply:
It has been fixed. It doesn't exist every time.